common/unsafe/src/main/java/org/apache/spark/unsafe/Platform.java

```diff
@@ -304,7 +304,7 @@ public static void throwException(Throwable t) {
   static {
     boolean _unaligned;
     String arch = System.getProperty("os.arch", "");
-    if (arch.equals("ppc64le") || arch.equals("ppc64")) {
+    if (arch.equals("ppc64le") || arch.equals("ppc64") || arch.matches("s390x")) {
       // Since java.nio.Bits.unaligned() doesn't return true on ppc (See JDK-8165231), but
       // ppc64 and ppc64le support it
```

Comment:
Not big deal, but if `s390x` is listed here because of the same reason (`java.nio.Bits.unaligned() doesn't return true on`...), to update this comment seems good.

Reply:
done changed .matches to .contains

Reply:
@ketank-new Is this comment to address line 309 instead of line 307?

Reply:
I think the question is whether the comment also applies to s390x. If not, I think it's OK to leave. But I guess the other question is, are you saying `Bits.unaligned()` also doesn't return the correct value in some cases on this platform? if it works then this isn't so necessary. If we're not sure, OK to force it here I think.

Reply:
Yes, if it is not, ok to leave.   

According to my understanding, [this comment](https://github.com/apache/spark/pull/24861/files#r294620569) did not seem to answer the original question.